cmake/modules/AddSwift.cmake

```diff
@@ -1131,16 +1144,6 @@ function(_add_swift_library_single target name)
     set(PLIST_INFO_BUILD_VERSION)
   endif()
 
-  # On Linux and FreeBSD add the linker script that coalesces protocol
-  # conformance sections. This wouldn't be necessary if the link was done by
-  # the swift binary: rdar://problem/19007002
```

Comment:
This comment still applies.  We wouldn't need to specify the object files manually if we linked the sources using `swift`.